src/guide/bash.md

```diff
@@ -504,34 +541,7 @@ sticking to English alphanumeric characters and underscores**.
 
 ## 6. Transferring assets
 
-After minting the assets, you can transfer them to another account. To be
```

Comment:
We should return this part at some point.

Reply:
I believe we should consider creating a separate section for describing the functional capabilities of working with permissions in client_cli.

Reply:
Since you've added https://github.com/hyperledger/iroha-2-docs/issues/342, is this conversation resolved?

Reply:
> Clear to merge.